partials/examples/masterdetail.html

```diff
@@ -0,0 +1,10 @@
+<p>
+    This example shows default master detail renderer.
+</p>
+<div class="example">
+    <listing-control schema="vm.schema" uischema="vm.uiSchema" data="vm.data"></listing-control>
+</div>
+<p>
+  The renderer tries to find a fitting ui schema for the detail. If there is none, one is generated, otherwise the registered ui shema is used.
```

Comment:
Use 'UI schema' instead of 'ui schema'. Also, I'd not mention the UiSchemaRegistry; describing that registering multiple UI schemas is possible is already sufficient.